### ISSUE TITLE
Discussion of session 5 readings: Christen 2017 & Vézina 2020

### ISSUE THREAD

**Issue by gabrielbodard** [SunoikisisDC/SunoikisisDC-2020-2021#27]
* Kimberly Christen, Alex Merril & Michael Wynne. 2017. “A Community of Relations: Mukurtu Hubs and Spokes.” _D-Lib Magazine_ 23.5/6, Available: https://doi.org/10.1045/may2017-christen 
* Brigitte Vézina. 2020. “Ensuring Respect for Indigenous Cultures: A Moral Rights Approach.” _CIGI Papers_ No. 243 (May 2020). Available: https://www.cigionline.org/sites/default/files/documents/vezina-paper_1.pdf

**Comment by PAZHames** [comment]
### A Community of Relations: Mukurtu Hubs and Spokes
Kimberly Christen, Alex Merrill and Michael Wynne (2017)

Mukurtu started as a grassroots project to display and curate the Warumungu Aboriginal community's digital materials 'using their own cultural protocols’. The importance of instating cultural protocols was seen when the Warumungu were initially distressed at seeing their digital materials online. Some reasons for this were the display of images of the deceased with no warnings and pictures of scared sites with no connection to their context or ancestors. Christen ascertained that narrative layers needed to be included with individual objects and that they needed to control who had access to which objects. 

A Mukurtu is a ‘dilly bag’ – a safe space for elders to keep sacred objects. It must be opened up for the young to see, when they ask. This is what the archive aimed to be – a safe space for the communities sacred objects, through which they could share. Between 2002-2005 Christan collaborated with the Warumungu at three separate centres, laying the foundation of their collaborative work and the program was launched in 2007. It’s features were:

* 	Each object was tagged with a set of protocols regarding family relations, gender, and country affiliations
* 	Each user had an individual profile, which automatically matched them to the content they were allowed to see
* 	The platform was updatable to community needs over time

The project was then extended and the Washington State University Libraries formed a partnership with six tribes in the Plateau region. This brought in materials from already existing collections, such as the National Anthropological Archives (NAA), the National Museum of the American Indian (NMAI) at the Smithsonian Institution, and the Northwest Museum of Art and Culture (MAC) in Spokane, Washington. There were differential metadata requirements across fields and between communities and institutions. Rather than having the indigenous experience as a note on the institutional data, the system put native metadata and institutional metadata side by side; presenting them as two complimentary layers of the one artefact. Despite being alongside one another, the two were kept distinct, with tribal administrators and institutions each able to edit their own content but not the other’s. This links the histories of ‘collection and colonisation’ with that of ‘survival and adaptation’ and shows how the two are inherent in the same artefact. 

The platform then went from a custom project to a free, open-source management tool, as Christen became aware it would benefit other communities with similar concerns. It was made into a more stable and easily updatable tool. A particularly interesting update came in the development of Traditional Knowledge (TK) labels. As many native communities do not own their patrimony, these could be attached to open-source data, to make users aware of what the narrative layers of an object and who should have access to it, regardless of copyright. 

The project in its current iteration has developed into Mukurtu Hubs and Spokes. Hubs are regional support and training centres for the tribal archives, and the archives, libraries and museums are the spokes. One problem was that many Mukurtu users did not have the resources, infrastructure or programming skills to contribute in the way individuals often can to other open source platforms. To counteract this, the technical foundation of Mukurtu remained at WSU, which then provided additional support and outreach to the hubs. Those working at the hubs were trained centrally. The team are now planning on expanding the project even further and including more pre-existing collections. A key aspect of the ‘Mukurtu shared’ workflow will be that an object being added to the collection will not merely undergo technical checks, but also cultural and ethical ones. 

A few thoughts:

* 	I like the focus on the importance of the system being updatable to community needs over time. I felt it really drew attention to the community as a developing entity, rather than a set and immovable value system. 
* 	Although I absolutely see the practical requirements of limiting editing access to the community’s own uploads, I do wonder whether this presents a bit of an ethical conflict? Surely some of the artefacts in existing collections are just as sensitive and so could benefit from limited access? If the point of the project is to hand back control over their own patrimony to indigenous tribes, then should they not be given control over the whole lot? It seems like the system is still very much in an institutional framework. That said, I think this project is absolutely amazing and such a wonderful step in the right direction.

**Comment by alexandrabushman** [comment]
**Ensuring Respect for Indigenous Cultures: A Moral Rights Approach**
Brigitte Vézina (2020)

The article focuses on the recognition of copyright over traditional cultural expressions (TCE)s and the intellectual property (IP) protection for them, specifically when it comes to Indignous cultures. Attempts to apply copyright law to TCEs generally fail due to the conceptual difference between Western and Indigenous notions of cultural creativity. The article advocates for the adoption of a _sui generis_ model of protection that is based on the precepts of moral rights of copyright. This is geared towards preserving and respecting the reputation of Indigenious peoples in order to maintain the integrity of their culture and ensure proper attribution when their TCEs are used by others. The suis generis model comes from the scope of work carried out by the World Intellectual Property Organization (WIPO), the UN agency in charge of IP matters, but there is no legal instrument that has been internationally adopted to date. 

The WIPO describes TCEs as the ways that traditional cultures are expressed or manifested. TCEs are passed down from generation to generation and deeply linked to a community and its heritage (i.e. performances, ceremonies, music, dance, art, designs, names, symbols, handicrafts, folklore, legends, ect.). TCEs are extremely crucial to an Indigenous culture’s survival because they help shape the social and collective identity of a specific group and are intended as a tool to preserve and pass down a group’s living culture and traditions in accordance with their customary and traditional usage. TCEs are often under threats to their identity and cultural vitality caused by misappropriation and misuse. It is critical that Indigenous peoples have control over their TCEs to ensure any uses are respectful and according to their wishes.

The moral rights of copyright are debated from country to agency, however, the article addresses four main types of rights:
- the right of attribution (or paternity), the author has the right to claim authorship and to be recognized as the author of a work.
- the right of integrity (or respect), the author can prevent modifications of their work and object to derogatory treatment.
- the right of disclosure (or publication), the author alone may decide whether, when and how their work may be published or made available to the public.
- the right of withdrawal (or retraction or renunciation), the author can take back their work and prevent any further reproduction, distribution or representation.

TCE holders have the right to be fully, properly, and accurately acknowledged and named as the primary guardians and interpreters of their culture. TCEs often hold great cultural and spiritual significance so maintaining the overall integrity and context are fundamental for proper representation. If TCEs are used in offensive or derogatory ways, their meaning is misinterpreted and potentially lost. Sacredness is often invoked to justify an increased degree of protection to preserve their symbolic and private character. Some sacred and secret TCEs have been shared without their holders’ consent and have been misinterpreted and misrepresented. These rights are particularly relevant in the context of digitization and the online platform, Mukurtu, is one initiative that aims to help navigate the rights and interests of Indeginous communities and their TCEs in the digital world. However, Murkurtu does not have unanimous support from Indigenous peoples since TCEs are expressed and preserved according to traditions that may be executed and transmitted only within fixed parameters.

Copyright protection is only available to the limited number of TCEs that can meet the requirements. One requirement in particular is the originality criterion, but since TCEs are transmitted from generation to generation with a focus on their preservation, originality is not necessarily encouraged in the way that it is under Western creativity paradigms. There is also no clear author of TCEs, as they are collectively and communally generated which conflicts with copyright’s authorship principle. Lik authorship, the principle of ownership in copyright law is at odds with Indigenous artistic traditions as TCEs are usually held for the benefit of a community as a whole. Copyright laws also rely on the principle that works eventually fall into the public domain. This temporally finite protection and the concept of the public domain appear incompatible with the needs, wishes and aims of Indigenous peoples whose TCEs are rooted in many thousands of years of history and are meant to be further transmitted to generations to come.

The international community is equipped to make the necessary adjustments to create a suitable international _sui generis_ regime for the moral rights-like protection of TCEs. The WIPO Intergovernmental Committee on Intellectual Property and Genetic Resources, Traditional Knowledge and Folklore (IGC) is developing international standards and has arrived at the idea of a “tiered approach” to protection. With this approach, different kinds or levels of rights would be attached to TCEs, depending on a particular TCE’s nature and characteristics, the level of control retained by the beneficiaries over the TCE, and its degree of sacredness or secrecy. This approach offers an opportunity to reflect the IP system’s balance between the interests of right holders and the general public. The tiered approach implies that exclusive economic rights could be appropriate for some forms of TCEs (i.e. secret and/or sacred), whereas a moral rights-based model could be suitable for TCEs that are publicly available or widely known but still linked to specific holders. Applied with economic rights (i.e. trademark law and other IP laws), the tiered approach places great value on moral rights.

Under a _sui generis_ model, the key rationale in favour of protecting TCEs is the “guardianship” relationship and contrasted with ownership, which is a core concept in most IP law systems. Since TCEs might be shared, incorporated, and evolved along with different Indigenous groups that all identify and hold a guardianship relationship with them, procedures should be in place to facilitate cooperation and settle disputes. TCEs often encompass cultural elements that are integral to Indigenous peoples’ sense of identity and bear the distinct mark of their holders. As TCEs are collectively and communally held, the moral rights of Indigenous peoples must be communal. “Just as moral rights vest automatically in the author, so too would _sui generis_ moral rights vest in the community.” Moral rights would only regulate the relationship between the community and the outside world, not preventing use as a traditional and customary context.

When applying the right of integrity, to determine what is offensive should be done both subjectively, from the point of view of the community that claims violation, and objectively, by the court (within the guidelines developed legislatively or through case law and informed by Indigenous customary laws, practices, and protocols). Because of the intergenerational nature of TCEs and the fact that customary laws often intend that they are perpetually held, there is concern of submitting TCEs to the prospect of eventually entering the public domain. While protection should not necessarily be everlasting, protection should be linked to the group maintaining a “guardianship” with a TCE. TCE holders should also be able to rely on several enforceable means of compensation for the harm suffered and to prevent any recurrence of the infringing activities (i.e. official recognition of the violation, damages, an injunction to make the harm stop, an order for a public apology or non-monetary compensation).

Granting of moral rights in TCEs is a prodigious leap in the right direction, but is faced with strong opposition and polarizing fundamental debates. _Sui generis_ protection is bound to dramatically change the status quo of how the world has grown accustomed to interacting with TCEs and is likely to force many actors in various sectors of the economy, and in society in general, to radically change their behaviour.

My thoughts:
- As far as copyright and moral rights are concerned, I think any digital cultural heritage should be solely run and dependent on the Indigenous community. In other words, I think that these communities have the right to define their TCEs and digitally publicize their cultures in the way that they deem fit. 
- I was surprised that there was not any agreed upon, internationally-set of standards that protect TCEs that are put in place yet. Copyright laws, or something like them (like the proposed suis generis model) could protect TCEs from cultural appropriation and entitle them to financial compensation and damages.

**Comment by nicolealexandra33** [comment]
Both super interesting articles, and I am really happy to see some progress done on protecting indigenous culture and the decolonisation of history and archaeology. Just a few thoughts which aren't directly from the readings, but they did make me think:

- I have been seeing a lot more indigenous youth here in North America who are rising in popularity on various social media platforms. In their content, they frequently share some information about their culture and even sometimes are in traditional dress. While I believe the articles were maybe discussing discussing more specific and religious (and sometimes ancient) artefacts and limiting their digital access to community members rather than open to the general public (particularly Christen), I wonder how this may change or what other ethical issues may arise as the younger generations of indigenous and Aboriginal peoples become more active on various social media platforms which are often quite hard to control how they are shared. I think it's great that there are more indigenous 'influencers', and I have certainly learned a lot through them, but I do wonder when the particular issues these articles discuss and their social media content will end up intersecting and how that will be resolved
- Again reflecting on the idea of limiting access of certain cultural information to certain members vs. open to the public, I know a lot of people whose families have immigrated from their countries/regions of origin (or who were adopted), but the younger 'first generation' feel a bit disconnected from their cultural heritage especially as they did not grow up in the region or have a shared lived experience with many of their peers from the same heritage (ex.: not having grown up on an indigenous reservation, etc.). I wonder if there would be a way for these people who were genuinely interested and willing to devote themselves to learning more about their heritage to 'apply' to such digital communities, so that they could access certain protected information especially if they live quite far from their region/nation of heritage and cannot access it easily (especially during a pandemic, but also ex.: a bit hard for some people of Pacific Islander descent living in Europe or North America to get there physically)

**Comment by ChantalvanEgdom** [comment]
Thank you for the summary @PAZHames ! I think the seperation between the layers is a good step and seems like a great way to maintain or even hand back the communities' control over their own cultural heritage. I agree with you, that limiting the communities' access to their own material seems like a step back/into the wrong direction... Probably ties in with the bigger problem of who ultimately is in control of the collections, which is an issue that could do with some reallocation, but the project does seem to set a great first step in the right direction!

**Comment by ChantalvanEgdom** [comment]
@alexandrabushman I completely agree with your thoughts; the Indigenous communties and their wishes should always be prioritised over the interests, how well-intended they may be, of "outsiders" concerning Indigenous material. The _sui generis_ model seems like a good solution. 

To add to @nicolealexandra33 's point on indigenous material gaining popularity on social media; I have seen this too and the posted material has been so great and informative! Something that I do see happening is whenever material is shared without credit, there is often somebody in the comment section who will draw the attention to the fact that the credits belong to account X, which often results in the credits being added later on. It'd be great if there was a way of ensuring the use of correct credits and more importantly, the permission to share being needed but I guess that would be difficult to enforce...

**Comment by K-Doering** [comment]
>Again reflecting on the idea of limiting access of certain cultural information to certain members vs. open to the public, I know a lot of people whose families have immigrated from their countries/regions of origin (or who were adopted), but the younger 'first generation' feel a bit disconnected from their cultural heritage especially as they did not grow up in the region or have a shared lived experience with many of their peers from the same heritage (ex.: not having grown up on an indigenous reservation, etc.). I wonder if there would be a way for these people who were genuinely interested and willing to devote themselves to learning more about their heritage to 'apply' to such digital communities, so that they could access certain protected information especially if they live quite far from their region/nation of heritage and cannot access it easily (especially during a pandemic, but also ex.: a bit hard for some people of Pacific Islander descent living in Europe or North America to get there physically)
>
Thanks to both for such great summaries! @nicolealexandra33, I was also thinking about the access question in terms of those with genuine and/or research-driven interest and I think "applying" would be a remedy for those folks. The gatekeepers could then make a decision on how to handle their access. As someone who admittedly knows very little about any indigenous culture, the idea that women are prohibited from having access to certain aspects of the database gave me an immediate negative reaction. While I want to remain sensitive to the cultural mores of the groups I also think that the access should not be technically regulated but instead be socially regulated. If each member of the group knows what is appropriate to be seen by each sex then I don't think the electronic blocking is necessary. I would love to hear other thoughts about this.

**Comment by K-Doering** [comment]
>Although I absolutely see the practical requirements of limiting editing access to the community’s own uploads, I do wonder whether this presents a bit of an ethical conflict? Surely some of the artefacts in existing collections are just as sensitive and so could benefit from limited access? If the point of the project is to hand back control over their own patrimony to indigenous tribes, then should they not be given control over the whole lot? It seems like the system is still very much in an institutional framework. That said, I think this project is absolutely amazing and such a wonderful step in the right direction.
>
Agreed @PAZHames, thank you for your comments! I think that the best next step would be to fund a position or fellowship for members of the six tribes to take up residency at WSU and to be partner gatekeepers. There is something rather backwards about the thought of a white man deciding what is best and most constructive to be included in the databases. To echo your last point though, I think this is absolutely headed in the right direction and I don't want to downplay the great work that has been undertaken so far.

**Comment by FabioDFernandes** [comment]
Thank you both for your super detailed summaries!

> > Again reflecting on the idea of limiting access of certain cultural information to certain members vs. open to the public, I know a lot of people whose families have immigrated from their countries/regions of origin (or who were adopted), but the younger 'first generation' feel a bit disconnected from their cultural heritage especially as they did not grow up in the region or have a shared lived experience with many of their peers from the same heritage (ex.: not having grown up on an indigenous reservation, etc.). I wonder if there would be a way for these people who were genuinely interested and willing to devote themselves to learning more about their heritage to 'apply' to such digital communities, so that they could access certain protected information especially if they live quite far from their region/nation of heritage and cannot access it easily (especially during a pandemic, but also ex.: a bit hard for some people of Pacific Islander descent living in Europe or North America to get there physically)
> 
> Thanks to both for such great summaries! @nicolealexandra33, I was also thinking about the access question in terms of those with genuine and/or research-driven interest and I think "applying" would be a remedy for those folks. The gatekeepers could then make a decision on how to handle their access. As someone who admittedly knows very little about any indigenous culture, the idea that women are prohibited from having access to certain aspects of the database gave me an immediate negative reaction. While I want to remain sensitive to the cultural mores of the groups I also think that the access should not be technically regulated but instead be socially regulated. If each member of the group knows what is appropriate to be seen by each sex then I don't think the electronic blocking is necessary. I would love to hear other thoughts about this.

It's a good point you made, Kéyah, that for those who wish to 'apply' to contribute, there could be a decision made by the gatekeepers for their admission. The example @nicolealexandra33 alluded to of 'first generation' descendants of these communities made me think of what Phoebe said about the community as a 'developing entity' - which all communities are, we are not relics in time. I would think their input would be valuable in a different way, in that they are still likely culturally connected to some extent but can express this particular evolution of theirs through these systems, and they should be allowed to have access to their origins too. Diasporas maintaining strong cultural ties with their mother communities/lands are a thing after all across the world with so many cultures. 
Indeed, at least to us, the prohibition of women from certain aspects evokes an immediate negative reaction, though I also know very little of indigenous/aboriginal cultures and so do not feel like I can comment on that. But I agree with you that 'social' regulation over 'technical' is a better way to go - a more personal approach to these projects is always better as it allows for greater fluidity when necessary.

**Comment by FabioDFernandes** [comment]
> > Although I absolutely see the practical requirements of limiting editing access to the community’s own uploads, I do wonder whether this presents a bit of an ethical conflict? Surely some of the artefacts in existing collections are just as sensitive and so could benefit from limited access? If the point of the project is to hand back control over their own patrimony to indigenous tribes, then should they not be given control over the whole lot? It seems like the system is still very much in an institutional framework. That said, I think this project is absolutely amazing and such a wonderful step in the right direction.
> 
> Agreed @PAZHames, thank you for your comments! I think that the best next step would be to fund a position or fellowship for members of the six tribes to take up residency at WSU and to be partner gatekeepers. There is something rather backwards about the thought of a white man deciding what is best and most constructive to be included in the databases. To echo your last point though, I think this is absolutely headed in the right direction and I don't want to downplay the great work that has been undertaken so far.

Agreed!

**Comment by nicolealexandra33** [comment]
@ChantalvanEgdom Ah that's great to hear that viewers/followers mention or give credit to these content creators especially if it is not given immediately. While we've agreed that some of these platforms are hard to monitor and control how the content is shared and there are no real legal sanctions or boundaries at this point, it's nice to know that online communities who engage with this contact are generally looking out for the creator's best interests

**Comment by Jessamyn-kcl** [comment]
Thanks for the great summaries guys! I think the Christen et al article particularly raises some very interesting issues: it's definitely great to see institutions engaging with indigenous communities and appropriate display. Displaying the dead especially is an area where lots of museums perhaps aren't as diligent as they should be in when it comes to ensuring cultural beliefs and appropriate warnings are honoured (think about how many museums unwrap and display mummies etc) so it's definitely good to see museums taking a more active stance in that. Similarly, I like the idea of the two sets of information being available, both the museum's context and the indigenous context, because I think that does hold the institution more accountable to keeping its own information up to date and accurate. That all being said, I do agree with the concerns raised by several people up-thread who have pointed out issues of gender restrictions, or restrictions based on knowing detailed information about your ancestors limiting access, and preventing people who are trying to become more connected with their own past - though presumably there would be an element of discretion here on the part of those controlling access.

**Comment by despinaborcea** [comment]
> @alexandrabushman I completely agree with your thoughts; the Indigenous communties and their wishes should always be prioritised over the interests, how well-intended they may be, of "outsiders" concerning Indigenous material. The _sui generis_ model seems like a good solution.
> 
> To add to @nicolealexandra33 's point on indigenous material gaining popularity on social media; I have seen this too and the posted material has been so great and informative! Something that I do see happening is whenever material is shared without credit, there is often somebody in the comment section who will draw the attention to the fact that the credits belong to account X, which often results in the credits being added later on. It'd be great if there was a way of ensuring the use of correct credits and more importantly, the permission to share being needed but I guess that would be difficult to enforce...

Thank you both for your summaries! I agree with Nicole and Chantal's point about cultural heritage and social media. Sometimes it is difficult for the viewer to draw a distinction between the indigenous groups and the content of the 'influencers' with credits, because the latter might not name/obtain permission from the former. I can think of a slightly different example from Romania - traditional folk costumes of the people living in some rural areas (of whom some still produce the clothes manually, like they did centuries ago) have been repeatedly copied in design/patterns/etc by mammoth high fashion clothing brands without offering any credit. This is not an isolated case and the issue is that, despite repeated lawsuits, nothing much has changed. In this sense, the double-layered TCE methods discussed by Brigitte Vézina is useful for other contexts as well, as it could result in a more ethical approach to the promotion of (indigenous) cultural heritage.

**Comment by despinaborcea** [comment]
> Thank you both for your super detailed summaries!
> 
> > > Again reflecting on the idea of limiting access of certain cultural information to certain members vs. open to the public, I know a lot of people whose families have immigrated from their countries/regions of origin (or who were adopted), but the younger 'first generation' feel a bit disconnected from their cultural heritage especially as they did not grow up in the region or have a shared lived experience with many of their peers from the same heritage (ex.: not having grown up on an indigenous reservation, etc.). I wonder if there would be a way for these people who were genuinely interested and willing to devote themselves to learning more about their heritage to 'apply' to such digital communities, so that they could access certain protected information especially if they live quite far from their region/nation of heritage and cannot access it easily (especially during a pandemic, but also ex.: a bit hard for some people of Pacific Islander descent living in Europe or North America to get there physically)
> > 
> > 
> > Thanks to both for such great summaries! @nicolealexandra33, I was also thinking about the access question in terms of those with genuine and/or research-driven interest and I think "applying" would be a remedy for those folks. The gatekeepers could then make a decision on how to handle their access. As someone who admittedly knows very little about any indigenous culture, the idea that women are prohibited from having access to certain aspects of the database gave me an immediate negative reaction. While I want to remain sensitive to the cultural mores of the groups I also think that the access should not be technically regulated but instead be socially regulated. If each member of the group knows what is appropriate to be seen by each sex then I don't think the electronic blocking is necessary. I would love to hear other thoughts about this.
> 
> It's a good point you made, Kéyah, that for those who wish to 'apply' to contribute, there could be a decision made by the gatekeepers for their admission. The example @nicolealexandra33 alluded to of 'first generation' descendants of these communities made me think of what Phoebe said about the community as a 'developing entity' - which all communities are, we are not relics in time. I would think their input would be valuable in a different way, in that they are still likely culturally connected to some extent but can express this particular evolution of theirs through these systems, and they should be allowed to have access to their origins too. Diasporas maintaining strong cultural ties with their mother communities/lands are a thing after all across the world with so many cultures.
> Indeed, at least to us, the prohibition of women from certain aspects evokes an immediate negative reaction, though I also know very little of indigenous/aboriginal cultures and so do not feel like I can comment on that. But I agree with you that 'social' regulation over 'technical' is a better way to go - a more personal approach to these projects is always better as it allows for greater fluidity when necessary.

I completely agree, Fabio! I think it would be quite interesting and useful to make available the 'external' perceptions of a diaspora on their source community. This reminds me of the GB1900 gazetteer creation a little - I think a similar layer of 'diaspora memories' applied to a digital map could reflect perceptions on those communities well. While the main and perhaps most reliable sources of information about that community would remain the facts sourced by that group, sometimes the impressions/memories of the diaspora can help illuminate some of the strongest perceptions (even biases and prejudices) regarding their homeland.

**Comment by FabioDFernandes** [comment]
> > Thank you both for your super detailed summaries!
> > > > Again reflecting on the idea of limiting access of certain cultural information to certain members vs. open to the public, I know a lot of people whose families have immigrated from their countries/regions of origin (or who were adopted), but the younger 'first generation' feel a bit disconnected from their cultural heritage especially as they did not grow up in the region or have a shared lived experience with many of their peers from the same heritage (ex.: not having grown up on an indigenous reservation, etc.). I wonder if there would be a way for these people who were genuinely interested and willing to devote themselves to learning more about their heritage to 'apply' to such digital communities, so that they could access certain protected information especially if they live quite far from their region/nation of heritage and cannot access it easily (especially during a pandemic, but also ex.: a bit hard for some people of Pacific Islander descent living in Europe or North America to get there physically)
> > > 
> > > 
> > > Thanks to both for such great summaries! @nicolealexandra33, I was also thinking about the access question in terms of those with genuine and/or research-driven interest and I think "applying" would be a remedy for those folks. The gatekeepers could then make a decision on how to handle their access. As someone who admittedly knows very little about any indigenous culture, the idea that women are prohibited from having access to certain aspects of the database gave me an immediate negative reaction. While I want to remain sensitive to the cultural mores of the groups I also think that the access should not be technically regulated but instead be socially regulated. If each member of the group knows what is appropriate to be seen by each sex then I don't think the electronic blocking is necessary. I would love to hear other thoughts about this.
> > 
> > 
> > It's a good point you made, Kéyah, that for those who wish to 'apply' to contribute, there could be a decision made by the gatekeepers for their admission. The example @nicolealexandra33 alluded to of 'first generation' descendants of these communities made me think of what Phoebe said about the community as a 'developing entity' - which all communities are, we are not relics in time. I would think their input would be valuable in a different way, in that they are still likely culturally connected to some extent but can express this particular evolution of theirs through these systems, and they should be allowed to have access to their origins too. Diasporas maintaining strong cultural ties with their mother communities/lands are a thing after all across the world with so many cultures.
> > Indeed, at least to us, the prohibition of women from certain aspects evokes an immediate negative reaction, though I also know very little of indigenous/aboriginal cultures and so do not feel like I can comment on that. But I agree with you that 'social' regulation over 'technical' is a better way to go - a more personal approach to these projects is always better as it allows for greater fluidity when necessary.
> 
> I completely agree, Fabio! I think it would be quite interesting and useful to make available the 'external' perceptions of a diaspora on their source community. This reminds me of the GB1900 gazetteer creation a little - I think a similar layer of 'diaspora memories' applied to a digital map could reflect perceptions on those communities well. While the main and perhaps most reliable sources of information about that community would remain the facts sourced by that group, sometimes the impressions/memories of the diaspora can help illuminate some of the strongest perceptions (even biases and prejudices) regarding their homeland.

That would certainly be a good idea, Despina! And I believe you are correct in all you say. Perceptions even within communities are often diverse.
